dev-tools/action-install-minikube.sh

```diff
@@ -24,26 +29,55 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]; then
         # Update permissions for .kube and .minikube
         sudo chown -R $USER $HOME/.kube $HOME/.minikube
 
+        export IP=$(minikube ip)
```

Comment:
should use `fluvio cluster set-minikube-context`

Reply:
Tried this already, this step is required, otherwise it will fail to find the cluster. 

This step is also run in the `action-install-fluvio-cluster.sh` file.

Reply:
so this is not show stopper but we should consolidate this into fluvio.   @vijaylaxmid @nicholastmosher let's create comprehensive ticket for `set-minikube-context` that should work anywhere